CONTRIBUTING.md

```diff
@@ -3,275 +3,16 @@ Contributing to the Flutter engine
 
 [![Build Status](https://cirrus-ci.com/flutter/engine.svg)](https://cirrus-ci.com/flutter/engine)
 
-_See also: [Flutter's code of conduct](https://flutter.io/design-principles/#code-of-conduct)_
+_See also: [Flutter's code of conduct](https://github.com/flutter/flutter/blob/master/CODE_OF_CONDUCT.md)_
 
 Welcome
 -------
 
-This guide introduces you to building and contributing to the Flutter engine.
-For an introduction to contributing to the Flutter framework, see [the equivalent
-document in the framework's repository](https://github.com/flutter/flutter/blob/master/CONTRIBUTING.md).
+For an introduction to contributing to Flutter, see [our
+contributor
+guide](https://github.com/flutter/flutter/blob/master/CONTRIBUTING.md).
```

Comment:
I don't know if there's another PR - but that document links back to the "engine wiki", which currently redirects to the engine landing page.

Reply:
yeah that gets fixed by https://github.com/flutter/flutter/pull/23963